### DISCUSSION TITLE
Github now defaults the "clone" button to the code spaces tab, and it is driving me insane

### DISCUSSION THREAD

**Discussion by arvid-norlander** [community/community#60039] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

This tab is now default when I press the button to get the clone URL. It used to be the local tab that was default, and it still is on my personal account (as opposed to my work account, which is not attached to any organization, so that isn't it).

I don't use code spaces. I *don't want* to use code spaces. How do I change it back so the "Local" tab is the default?

![image](https://github.com/community/community/assets/132438847/ba985ae2-8fcd-41be-8eb1-c0fc4a6f9856)

All I want is the HTTPS URI so I can run `git clone`. I don't want github trying to *force feed* me their cloud offerings! I don't even ever want to use VSCode, I prefer emacs.

**Comment by GhostOf0days** [comment]
This isn't a solution, but just so you now, here are defaults for cloning. You could probably just copy paste from the top of your tab and add .git at end:

HTTPS: https://github.com/repoOwnerName/repoName.git (probably what you use)
SSH: git@github.com:repoOwnerName/repoName.git
Github CLI: gh repo clone repoOwnerName/repoName
